src/qproc/testChunkSpec.cc

```diff
@@ -115,7 +116,7 @@ BOOST_AUTO_TEST_CASE(Intersect) {
         c3.subChunks.push_back(i);
     }
     // Throws because chunkIds are different.
-    BOOST_CHECK_THROW(c1c2 = c1.intersect(c2), std::logic_error);
+    BOOST_CHECK_THROW(c1c2 = c1.intersect(c2), lsst::qserv::util::Bug);
```

Comment:
util::Bug is based on util::Issue, which is based on std::exception, not std::logic_error. The unit test fails because std::logic_error is no longer being thrown. The fact that util::Bug contains ERR_LOC made this failure much easier to find.

Reply:
👍